src/pages/Account/Login.tsx

```diff
@@ -32,6 +52,22 @@ export const Login: FC<LoginProps> = ({ formType = "initial" }) => {
         <img src="/sparkle-header.png" alt="" width="100%" />
       </div>
       <div className="auth-form-container">
+        {hasAlternativeLogins && (
+          <div className="Login__login-box">
+            <span>Quick log in with</span>
+
+            <div className="Login__alternative-logins">
+              {hasSamlAuthProviderId && (
```

Comment:
You are already checking for `hasAlternativeLogins` above and they have the same value, looks like this check here is redundant.

Reply:
As per https://github.com/sparkletown/sparkle/pull/1546#discussion_r652831141, it is the intended flow